ishell.go

```diff
@@ -495,31 +496,44 @@ func (s *Shell) multiChoice(options []string, text string, init []int, multiResu
 				selected = toggle(selected, cur)
 			}
 		}
-		update()
+		select {
```

Comment:
Using `select` does not change the behaviour. It will still be blocked since it is the only condition in the `select` statement.